packages/components/menu/src/menu-item.scss

```diff
@@ -0,0 +1,97 @@
+:host {
+  --_background: var(--sl-color-select-item-background);
+  --_border-radius: var(--sl-border-radius-popover-default);
+  --_cursor: pointer;
+  --_focus-outline: var(--sl-color-focusring-default) solid var(--sl-border-width-focusring-default);
+  --_gap: 0.25rem;
+  --_indent: calc(var(--sl-menu-item-indent, 0) * 26px + 6px);
+  --_gap-shortcut: 0.5rem;
+  --_gap-sub-menu-indicator: 1rem;
+  --_padding: 6px;
+  --_submenu-offset: -5px;
+}
+
+:host {
+  display: block;
+  flex-grow: 1;
+}
+
+:host(:hover) {
+  position: relative;
+
+  .safe-triangle {
+    display: block;
+  }
+}
+
+:host(:focus-visible) {
+  outline: var(--_focus-outline);
```

Comment:
Outline and hover next to each other looks a little bit weird. I had similar problem in tabs and that's why I manipulated with higher z-index there.


<img width="268" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/6d35f6b1-62b6-4697-935c-2888548b5631">

Reply:
Fixed